test/libponyc/util.cc

```diff
@@ -246,6 +246,11 @@ void PassTest::SetUp()
   opt.check_tree = true;
   opt.verify = true;
   opt.allow_test_symbols = true;
+#if defined(__pic__) || defined(__PIC__) || defined(__pie__) || defined(__PIE__)
```

Comment:
Should this have an `else` clause that sets `opt.pic = false;` just in case pass_opt_init defaults it to `true`?

Reply:
`pass_opt_init` doesn't seem to modify `pic` so I'm assuming you're referring to `ponyc_opt_process`. This function isn't used by the compiler tests so there is no problem here.

However, there seem to be some issues with these defines on the CI environment. I'll investigate that.

Reply:
IIRC, pass_opt_init memsets everything to 0 then initializes other fields. What you're doing I don't think it's a bug now, but if pass_opt_init were to change it could be. So I don't think this is the CI problem.